src/test/java/APIAssignment/APIAssignmentGet.java

```diff
@@ -0,0 +1,36 @@
+package APIAssignment;
+
+import io.restassured.*;
+import io.restassured.http.*;
+import io.restassured.path.json.*;
+import io.restassured.response.*;
+import static io.restassured.RestAssured.*;
+import io.restassured.specification.*;
+import org.junit.*;
+
+public class APIAssignmentGet {
+    @Test
+    public void getAllProductList() {
+        var productListURL = "https://automationexercise.com/api/productsList";
+        Response responseVariable = get(productListURL);
+        // validate status code
+        var statusCodeValue = responseVariable.getStatusCode();
+        Assert.assertEquals(200, statusCodeValue);
+        System.out.println("status code is "+statusCodeValue);
+        //validate content of list
+        RestAssured.baseURI = productListURL;
+        RequestSpecification specification = RestAssured.given();
+        Response response = specification.request(Method.GET);
```

Comment:
You can add Query or path parameter to get the particular response  on line no.23